thinc/config.py

```diff
@@ -97,12 +97,13 @@ def __init__(
         config: Union[Config, Dict[str, Dict[str, Any]]],
         errors: List[Dict[str, Any]],
         message: str = "Config validation error",
+        element: str = "",
     ) -> None:
         """Custom error for validating configs."""
         data = []
         for error in errors:
             err_loc = " -> ".join([str(p) for p in error.get("loc", [])])
-            data.append((err_loc, error.get("msg")))
+            data.append((element, err_loc, error.get("msg")))
```

Comment:
We should probably do an `if element:` / `else:` here so we don't add an empty string column (plus column spacing) if the error is raised with `element` unset.

Reply:
mypy is not happy about this solution ...

Reply:
We can probably get that sorted somehow, or the other option is to add the origin to the last column of the table instead of the first. Then the emptyness at the end wouldn't matter in cases when `element` is empty ?

Reply:
Ah okay, nevermind. It's not that big of a deal. I'll go ahead and merge this, and see if maybe I can find a way to solve this.